nixos/modules/services/mail/zeyple.nix

```
@@ -0,0 +1,91 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.zeyple;
+  bool2int = x: if x then "1" else "0";
+
+  gpgHome = pkgs.runCommand "zeyple-gpg-home" { } ''
+    mkdir -p $out
+    for file in ${lib.concatStringsSep " " cfg.keys}; do
+      ${pkgs.gnupg}/bin/gpg --homedir="$out" --import "$file"
+    done
+
+    # Remove socket files
+    rm -f $out/S.gpg-agent*
+  '';
+in {
+  options.services.zeyple = {
+    enable = mkEnableOption "Zeyple, an utility program to automatically encrypt outgoing emails with GPG";
+
+    user = mkOption {
+      type = types.str;
+      default = "zeyple";
+      description = "User to run Zeyple as.";
```

See explanation below.

```suggestion
      description = ''
        User to run Zeyple as.
        
        <note><para>
          If left as the default value this user will automatically be created
          on system activation, otherwise the sysadmin is responsible for
          ensuring the user exists.
        </para></note>
      '';
```